ascent/examples/fizz_buzz.rs

```diff
@@ -0,0 +1,93 @@
+//! FizzBuzz
+
+use ascent::ascent;
+
+ascent! {
+    // Facts:
+
+    relation number(isize);
+    
+    // Rules:
+    
+    relation divisible(isize, isize);
+
+    divisible(x, 3) <-- number(x) if x % 3 == 0;
+    divisible(x, 5) <-- number(x) if x % 5 == 0;
+
+    relation fizz(isize);
+
+    fizz(x) <-- number(x), divisible(x, 3), !divisible(x, 5);
+
+    relation buzz(isize);
+
+    buzz(x) <-- number(x), !divisible(x, 3), divisible(x, 5);
+
+    relation fizz_buzz(isize);
+
+    fizz_buzz(x) <-- number(x), divisible(x, 3), divisible(x, 5);
+
+    relation other(isize);
+
+    other(x) <-- number(x), !divisible(x, 3), !divisible(x, 5);
+}
+
+fn main() {
+    let mut prog = AscentProgram::default();
+    
+    prog.number = vec![
```

Comment:
Can we generate this from a range instead?

Reply:
Sure!

I'm not sure if FizzBuzz (and this specific implementation in particular) is much of a great example for ascent to begin with, but since it's a well-known yet simple "algorithm" I figured why not include it.

(I also tried implementing the [Sieve of Eratosthenes](https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes) but failed miserably. 😅)

Reply:
Yeah, Sieve of Eratosthenes doesn't look like a good fit for Datalog (it may be doable with lattices, but I don't think it would end up being a good example).

BTW, you may have forgotten to generate `prog.number` from a range here!

Reply:
> BTW, you may have forgotten to generate `prog.number` from a range here!

I did! Fixed.